bin/setup_venv.sh

```diff
@@ -176,15 +176,14 @@ source "${VIRTUAL_ENV}/bin/activate"
 
 set -o nounset # abort on unbound variable
 
-# Use the latest version of pip
+# Use the latest version of pip and pipenv
 pip3 install --upgrade pip
+pip install pipenv
 
-# Installs wheel package management, so that pulumi requirements install quickly
-pip3 install wheel
-
-# Get nodeenv version so that node can be installed before we install Python
-# dependencies because pulumi_eks depends on the presence of node.
-pip3 install "$(grep nodeenv "${script_dir}/../pulumi/python/requirements.txt")"
+# Install certain utility packages like `nodeenv` and `wheel` that aid
+# in the installation of other build tools and dependencies
+# required by the other python packages.
+PIPENV_VERBOSITY=-1 PIPENV_PIPFILE="${script_dir}/../pulumi/python/Pipfile" pipenv install --dev
```

Comment:
**annotation:**
Here we are:

1. Silencing an informational message telling us that pipenv is going to use the current venv with `PIPENV_VERBOSITY=-1`
2. Pointing `pipenv` at our `Pipfile` which is not in the current dir with `PIPENV_PIPFILE`
3. Only installing `dev` depenencies which are currently `nodeenv` and `wheel`